File: Uri-Online-Judge/Wellington/Python/Iniciante/1045.py
# -*- coding: utf-8 -*-

valores = raw_input().split(' ')
valores[0] = float(valores[0]) #converte valores
valores[1] = float(valores[1])
valores[2] = float(valores[2])
valores.sort(reverse=True) #ordena em ordem decrescente

if valores[0] >= valores[1] + valores[2]: #condições
	print('NAO FORMA TRIANGULO')
else:
	if valores[0] ** 2 == valores[1] ** 2 + valores[2] ** 2:
		print('TRIANGULO RETANGULO')
	if valores[0] ** 2 > valores[1] ** 2 + valores[2] ** 2:
		print('TRIANGULO OBTUSANGULO')
	if valores[0] ** 2 < valores[1] ** 2 + valores[2] ** 2:
		print('TRIANGULO ACUTANGULO')
	if valores[0] == valores[1] and valores[0] == valores[2]:
		print('TRIANGULO EQUILATERO')
	if valores[0] == valores[1] and valores[0] != valores[2] or valores[0] == valores[2] and valores[0] != valores[1] or valores[1] == valores[2] and valores[1] != valores[0]:
		print('TRIANGULO ISOSCELES')

File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1045.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

void ordenar(double array[], int tamanho){
    for (int i = 0; i < tamanho; i++) {
        for (int j = 0; j < tamanho; j++) {
            if (array[j]<array[j+1]) {
                int aux = array[j];
                array[j] = array[j+1];
                array[j+1] = aux;
            }
        }
    }
}

int main(){
    int tamanho = 3;
    double v[tamanho];
    
    for (int i = 0; i < tamanho; i++) {
        scanf("%lf", &v[i]);
    }
    
    ordenar(v, tamanho);
    
    if (v[0]>= v[1]+v[2]) {
        printf("NAO FORMA TRIANGULO\n");
    } else {
		 if (pow(v[0],2) == (pow(v[1],2) + pow(v[2],2))) {
	        printf("TRIANGULO RETANGULO\n");
	    }
	    if (pow(v[0],2) > pow(v[1],2) + pow(v[2],2)){
	        printf("TRIANGULO OBTUSANGULO\n");
	    }
	    if (pow(v[0],2) < pow(v[1],2) + pow(v[2],2)){
	        printf("TRIANGULO ACUTANGULO\n");
	    }
	    if (v[0] == v[1] && v[0] == v[2]){
	        printf("TRIANGULO EQUILATERO\n");
	    }
	    if (((v[0] == v[1] && v[0] != v[2]) || (v[0] == v[2] && v[0] != v[1])) || (v[1] == v[2] && v[1] != v[0])){
	        printf("TRIANGULO ISOSCELES\n");
	    }
	}
    
    return 0;
}
